DEVELOPMENT.md

```diff
@@ -115,3 +139,49 @@ bazel build third_party/chromium
 
 # Lastly, put the address returnd by the web server into the Chromium.
 ```
+
+#### Running webapp tests
+
+tl;dr. Use `ibazel test` for regular work (supports limiting the test amount with
+`fit/fdescript` and `console.log` usage) and `ibazel run` for karma console breakpoint
+debugging.
+
+We use karma testing chromium target for our UI unit tests in webapp. However,
+depending on the environments (OSS or internal TB.corp), the commands we use and
```

Comment:
I don't think you need to mention Tbcorp. It won't make sense to non-Googlers. In fact I think you can get rid of this entire paragraph.